framework/test/mocha/unit/modules/chain/transport/broadcaster.js

```diff
@@ -1,439 +0,0 @@
-/*
```

Comment:
Where are the replacement of these unit tests?

Reply:
since broadcaster is not exposed, it's not explicitly tested, but it's covered by the transport.spec test

Reply:
Some basic unit tests won't hurt anybody. Your call.